frontend/pages/menu.tsx

```diff
@@ -3,12 +3,42 @@ import { GetStaticProps } from "next";
 import { sanityClient } from "../lib/sanityClient";
 import { MenuItem } from "../types"; // Define your types if needed
 import MenuCard from '../components/menupage/MenuCard';
+import { useEffect} from 'react';
 
 interface MenuProps {
   menuItems: MenuItem[];
 }
 
 const MenuPage = ({ menuItems }: MenuProps) => {
+  const cartData0=[
+    {
+      _id: 'd3ca3b72-2418-4035-abfa-245f8696e8d1',
+      name: 'Roasted Duck',
+      description: 'This is the delicious rosted duck',
+      price: 67,
+      image: 'https://cdn.sanity.io/images/utvt9caf/test/dd236eafdebea0498587ac31fb102de59ddc2637-381x308.png',
+      quantity:1,
+    },
+    {
+      _id: 'd3ca3b72-2418-4035-abfa-245f8696e8d2',
+      name: 'Roasted Duck',
+      description: 'This is the delicious rosted duck',
+      price: 67,
+      image: 'https://cdn.sanity.io/images/utvt9caf/test/dd236eafdebea0498587ac31fb102de59ddc2637-381x308.png',
+      quantity:2,
+    },
+    {
+      _id: 'd3ca3b72-2418-4035-abfa-245f8696e8d3',
+      name: 'Roasted Duck',
+      description: 'This is the delicious rosted duck',
+      price: 67,
+      image: 'https://cdn.sanity.io/images/utvt9caf/test/dd236eafdebea0498587ac31fb102de59ddc2637-381x308.png',
+      quantity:3,
+    },
+  ];
+  useEffect(() => {
+    localStorage.setItem('cart', JSON.stringify(cartData0));
+  })
```

Comment:
All of our data comes from sanityIo and yes they would need to be set in localStorage but consider use useEffect with parameters, your useEffect is going to render every time the page loads or render

Reply:
And I have no idea why this useEffect needs to be here